halo/attest/keeper/keeper.go

```diff
@@ -789,7 +796,7 @@ func (k *Keeper) VerifyVoteExtension(ctx sdk.Context, req *abci.RequestVerifyVot
 		duplicate[vote.AttestHeader.ToXChain()] = true
 
 		// Ensure the votes are from the requesting validator itself.
-		if common.BytesToAddress(vote.Signature.ValidatorAddress) != ethAddr {
+		if !bytes.Equal(vote.Signature.ValidatorAddress, ethAddr[:]) {
```

Comment:
nit: `ethAddr.Cmp(vote.Signature.ValidatorAddress) != 0`

Reply:
I want to avoid casting `vote.Signature.ValidatorAddress` to a common.Address since it requires error checking. So I rather convert ethAddr to slice and compare that, so no error checks required.